packages/workbox-google-analytics/test/sw/index.js

```diff
@@ -158,11 +158,10 @@ describe(`initialize`, function() {
     sinon.spy(QueuePlugin.prototype, 'pushIntoQueue');
 
     googleAnalytics.initialize();
-    const payload = 'v=1&t=pageview&tid=UA-12345-1&cid=1&dp=%2F';
 
     self.dispatchEvent(new FetchEvent('fetch', {
       request: new Request(`https://${constants.URL.HOST}` +
-          `${constants.URL.COLLECT_PATH}?${payload}`, {
+          `${constants.URL.COLLECT_PATH}?${PAYLOAD}`, {
         method: 'GET',
       }),
     }));
```

Comment:
In general adding better comment descriptions to summarize what you are testing here would help. Especially if this is unclear from a read through.